app/views/datasets/_contact.html.erb

```diff
@@ -10,14 +17,8 @@
       </h3>
 
       <p>
-        <span><%= contact_name_for(dataset) %></span>
-        <span>
-          <% if contact_email_is_email?(dataset) %>
-            <%= mail_to(contact_email_for(dataset), nil, {'data-ga-event': 'contact', 'data-ga-publisher': @dataset.organisation.name}) %>
-          <% else %>
-            <%= link_to(contact_email_for(dataset), contact_email_for(dataset), {'data-ga-event': 'contact', 'data-ga-publisher': @dataset.organisation.name}) %>
-          <% end %>
-        </span>
+        <%= content_tag(:span, contact_name_for(dataset)) if contact_name_for(dataset) != "" %>
```

Comment:
Ruby has a `empty?` [method](https://ruby-doc.org/core-2.1.0/String.html#method-i-empty-3F) we could use here though it's more common place to use the `blank?` [method](http://api.rubyonrails.org/classes/Object.html#method-i-blank-3F) or the inverse `present?` method provided by Rails.

Using one of these makes the code a little more expressive:

```erb
<%= content_tag(:span, contact_name_for(dataset)) if contact_name_for(dataset).present? %>
```